### PR TITLE
Remove inactive ABtest entries

### PR DESCRIPTION
**Pull request by jessie-ross**
**This PR removes inactive ABTest entries:**

--- 

* newUsersWithFreePlan @razvanpapadopol 
* variantUserless @niranjan-uma-shankar 

These are both in `client/signup/controller.js`, I have commented them out as eslint was complaining.

I also ended up removing the synchronous wait for the `geo` data, this should improve user loading time for the moment.
When time comes to reimplement in ExPlat you will probably use `loadExperiment(experimentName).then(...)` or similar.

--- 

* reskinSignupFlow @niranjan-uma-shankar 

Reskin has been rerun in ExPlat.

---

* pageBuilderMVP 

Seems like this was already approved to be removed in #50510 but it didn't get merged by accident.

---

Part of the work on pbmo2S-Bv-p2 and under the issue 495-gh-Automattic/experimentation-platform

I am also hoping we get a nice decrease in bundle size from removing startup dependency on the `abtest` code.

#### Testing instructions

* Existing AT

### CODE REVIEW

File: client/signup/controller.js [hunk 1]
```diff
@@ -134,45 +131,50 @@ export default {
...
-						window.location = urlWithLocale;
-					} else {
-						next();
-					}
-				} )
-				.catch( () => {
-					removeWhiteBackground();
-					next();
-				} );
+			next();
```
Comment (jessie-ross) [inline client/signup/controller.js:134]: @niranjan-uma-shankar should there be a `removeWhiteBackground();` here too?

Comment (niranjan-uma-shankar) [reply client/signup/controller.js:134]: I haven't tested this PR yet but this looks good, `removeWhiteBackground()` is not necessary here.

**Review by yanirs — APPROVED** [review]
I haven't tested manually, but looks good to me based on a manual reference search and the assumption that the experiment owners will also have a look.

**Review by razvanpapadopol — APPROVED** [review]
Looks good and tests well! Always great to see bundle size decreasing 🎉

**Comment by tyxla** [comment]
Thank you for this one 🙌

**Pull request merged**